### PR TITLE
Improve PyPI metadata

### PR DESCRIPTION
**Pull request by dolfim-ibm**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+license = "MIT"
+readme = "README.md"
+homepage = "https://ibm.github.io/project-mognet/"
+repository = "https://github.com/IBM/project-mognet"
+documentation = "https://ibm.github.io/project-mognet/"
+classifiers = [
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Software Development :: Version Control :: Git",
+    "Development Status :: 5 - Production/Stable",
+    "Typing :: Typed"
```
Comment (RedRoserade) [inline pyproject.toml:15]: I think we should add a `py.typed` file for MyPy. See https://mypy.readthedocs.io/en/stable/installed_packages.html#creating-pep-561-compatible-packages

Comment (dolfim-ibm) [reply pyproject.toml:15]: just an empty file there?

Comment (RedRoserade) [reply pyproject.toml:15]: Yes, but you should check the generated wheel file to see if it's there. I've never done this myself.

Comment (dolfim-ibm) [reply pyproject.toml:15]: Done!

**Review by RedRoserade — APPROVED** [review]

**Pull request merged**